### DISCUSSION TITLE
[Getting YTDLError: aborted] When I host it to Heroku.

### DISCUSSION THREAD

**Discussion by SouvikUnscripted** [eritislami/evobot#791] [Q&A]
When I host it to Heroku, everything works fine, sound quality and everyone is alright. but sometimes I am getting YTDLError: aborted. How to fix the same?

**Comment by wolf-yuan-6115** [comment]
> When I host it to Heroku, everything works fine, sound quality and everyone is alright. but sometimes I am getting YTDLError: aborted. How to fix the same?

Can you provide full error message?
